hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoTransferHandler.java

```diff
@@ -84,7 +85,7 @@ public void preHandle(
         requireNonNull(tokenStore);
```

Comment:
Throughout this class I see a lot of warnings. Can we take care of them properly?